tests/source.c

```diff
@@ -185,8 +185,22 @@ void test_polycap_source_get_transmission_efficiencies() {
 
 	//This should work
 	polycap_clear_error(&error);
-	efficiencies = polycap_source_get_transmission_efficiencies(source, 1, 1, &energies, 5, NULL, &error);
+	efficiencies = polycap_source_get_transmission_efficiencies(source, 1, 7, energies, 5, NULL, &error);
 	assert(efficiencies != NULL);
+	polycap_transmission_efficiencies_free(efficiencies);
+
+	//Now we test actual values
+	//This will take a while...
+	polycap_clear_error(&error);
+	efficiencies = polycap_source_get_transmission_efficiencies(source, 1, 7, energies, 5000, NULL, &error);
+	assert(efficiencies != NULL);
+	assert(fabs(efficiencies->efficiencies[0] - 0.354) <= 0.005); //1 keV
```

Comment:
Uw referentie waarden: zijn die bekomen met Laszlo zijn versie?

Reply:
nee, met de mijne.
Zou het beter zijn met Laszlo's waarden? Denk niet dat het veel zou schelen, maar ben ook niet zeker of het niet meer dan 0.5% zou varieren :P

Reply:
Wel Laszlo zijn oude software is de enige referentie die we echt hebben denk ik: misschien best toch eens te bekijken of er geen significante verschillen zijn

Reply:
Kan het niet vergelijken met exact zelfde file als hierboven (enfin, zou kunnen, maar dan moet ik die prf en axs files etc nog maken) dus heb het maar getest met `xos1.inp`
Daar zijn mijn transmissie efficienties maximaal 1,7% hoger dan die van Laszlo, gemiddeld 1,3% denk ik.
Probleem is echter dat dit mss geen 100% eerlijk vergelijk is... Sinds Laszlo's versie heb ik Xraylib toegevoegd bvb, dus als die waarden iets variëren van Laszlo's libraries zitten we al met oplopende verschillen per reflectie. (Laszlo's fotonen reflecteren gemiddeld een klein beetje meer, maar dat is mss een effect van lekkende fotonen in Laszlo's code, die bij mij dus volledig geabsorbeerd worden door onzichtbare housing)

Hoe dan ook durf ik dus wel stellen dat onze waarden nog steeds dicht genoeg bij elkaar liggen. (en anders heb ik geen idee wat er moet veranderen om dezelfde waarden te bekomen :P )

Reply:
ok dan... ik wou u enkel wijzen op het belang van betrouwbare referentiewaarden 😄

Reply:
I know :) Wou het gewoon beargumenteren ;)

'Probleem' is ook dat ik niet denk dat Laszlo zijn polycap code ooit super experimenteel controleerde. Hij focuste vooral op capillairen (pun intended) en heeft me reeds meermaals gezegd dat het best kan dat er in zijn polycap code wat bugs zaten (bvb ivm reversibiliteit etc)